receiver/mongodbreceiver/metadata.yaml

```
@@ -1 +1,143 @@
 name: mongodbreceiver
+
+attributes:
+  database:
+    description: The name of a database.
+  memory_type:
+    description: The type of memory used.
+    enum:
+      - resident
+      - virtual
+      - mapped
+      - mapped_with_journal
+  operation:
+    description: The mongoDB operation being counted.
+    enum:
+      - insert
+      - query
+      - update
+      - delete
+      - getmore
+      - command
+  connection_type:
```

```suggestion
  connection_type:
    value: type
```

The `value` tag allows you to reuse a label. In this case `type` is sufficient to differentiate between connection types.